tests/unit/components/record/WebResources.spec.js

```diff
@@ -15,14 +15,14 @@ describe('components/record/WebResources', () => {
     const wrapper = factory();
     const props = {
       media: [
-        { rdfAbout: 'http://www.example.org/image.jpg', dcDescription: ['Art'] },
-        { rdfAbout: 'http://www.example.org/video.mp4', edmRights: { 'def': [ 'http://creativecommons.org/licenses/by-nc-sa/3.0/' ] } }
+        { about: 'http://www.example.org/image.jpg', dcDescription: ['Art'] },
+        { about: 'http://www.example.org/video.mp4', edmRights: { 'def': [ 'http://creativecommons.org/licenses/by-nc-sa/3.0/' ] } }
       ]
     };
 
     wrapper.setProps(props);
     for (let webResource of props.media) {
-      const webResourceBlock = wrapper.find('#' + cssesc(webResource.rdfAbout, { 'isIdentifier': true }));
+      const webResourceBlock = wrapper.find('#' + cssesc(webResource.about, { 'isIdentifier': true }));
```

Comment:
Template literals here instead?

Reply:
Do you mean `isIdentifier: true`?

Reply:
That may need excessive escaping due to the presence of { and } in the code.